app/views/events/choose_hotels_to_delete.html

```diff
@@ -0,0 +1,20 @@
+<% if @hotels.empty? %>
+<h1 id="header1">There is nothing to delete</h1>
+<% end %>
+
+<% @hotels.each do |hotel| %>
+<div class="col-sm-4">
+    <div class="panel">
+        <div class="panel-heading"><%= hotel.name %></div>
+        <div class="panel-body">
+            <%= link_to (image_tag hotel.picture.url(:medium)), event_delete_hotel_path(id: @event.id, hotel_id: hotel), method: :post, class:"img-responsive", id:"panel_img" %>
```

Comment:
Dałabym usuwanie przez osobnego buttona bo jakbym z tego korzystała to bym myślała, że po naciśnięciu na zdjęcie zostanę przeniesiona na stronę hotelu.